api/src/main/java/javax/enterprise/inject/spi/Annotated.java

```diff
@@ -94,6 +113,8 @@
     /**
      * <p>
      * Determine if the program element has an annotation of a certain annotation type.
+     *
+     * If the <tt>annotationType</tt> is repeatable, then returns true if at least one instance is available
```

Comment:
Sorry, I don't understand. My point is that `Annotated.isAnnotationPresent()` should behave as `java.lang.reflect.AnnotatedElement.isAnnotationPresent()`, i.e. return `false` for repeatable annotation.